### PR TITLE
Use a machine id for temporary display id

### PR DESCRIPTION
**Pull request by tejohnso**
@ahmedalsudani @fjvallarino Please review

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

File: installer/machine-id.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var os = require("os"),
+crypto = require("crypto"),
+totalMem = os.totalmem(),
+interfaces = os.networkInterfaces(),
+macs = "",
+cpus;
+
+cpus = os.cpus().map((cpu)=>{
```
Comment (ahmedalsudani) [inline installer/machine-id.js]: Does this method always return cpus in the same order?

File: installer/machine-id.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+var os = require("os"),
+crypto = require("crypto"),
+totalMem = os.totalmem(),
+interfaces = os.networkInterfaces(),
```
Comment (ahmedalsudani) [inline installer/machine-id.js]: @tejohnso same as the CPU question below, will we always get the interfaces in the same order?

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**